src/app/courses/courses.component.html

```diff
@@ -1 +1,8 @@
-<agmp-courses-list></agmp-courses-list>
+<div fxLayout="column" class="courses__container">
+  <agmp-courses-search></agmp-courses-search>
```

Comment:
I know it's one of best practises, but I would get rid of the feature component `agmp` prefix, like agmp-courses-search.
For shared components, like button or icon, it's nice to have.
But feature components hint that they are your own custom html elements.